### DISCUSSION TITLE
GDPR

### DISCUSSION THREAD

**Discussion by StefanUlbrich** [giscus/giscus#685] [Q&A]
Hi. Thanks for the amazing idea and project!

I wonder if I need should add a cookie/tracking consent banner to the website (personal blog) when using `giscus`. 
Isn't data about the user passed to GitHub by the script?

**Comment by laymonage** [comment]
Hey, we have a [privacy policy](https://github.com/giscus/giscus/blob/main/PRIVACY-POLICY.md) based on [the example given by gdpr.eu](https://gdpr.eu/privacy-notice). Please let me know whether that document answers your question and if there's anything I should add. Thanks!
